### PR TITLE
AOM-36: Remove the Clear Addon and upload button when the files is not selected

### PR DESCRIPTION
**Pull request by jomadoye**
# JIRA TICKET NAME:
AOM-36: [Remove the Clear Addon and upload button when the files is not selected](https://issues.openmrs.org/browse/AOM-36)

## SUMMARY:
This PR ensures that the `Clear Addon` and `upload Addon` button are not displayed when a file has not been selected

### CODE REVIEW

**Comment by dkayiwa** [comment]
Can you take a look at the merge conflicts on this?

File: app/js/components/homePage/manageApps/AddAddon.jsx [hunk 1]
```diff
@@ -9,41 +9,47 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export default function AddAddon({ handleClear, handleUpload }) {
+export default function AddAddon({ handleClear, handleUpload, displayManageOwaButtons, displayManageOwaButtonsState }) {
```
Comment (chibujax) [inline app/js/components/homePage/manageApps/AddAddon.jsx]: I think this is line is longer than 80 as specified in the guide, is this not important?

**Review by chibujax — COMMENTED** [review]
Looks good

File: app/js/components/homePage/manageApps/AddAddon.jsx [hunk 2]
```diff
@@ -9,41 +9,51 @@
...
+  const classesToUse = displayManageOwaButtonsState ?
+    'col-xs-6 col-sm-8' : 'col-xs-6 col-sm-12';
   return (
     <div className="container-fluid">
       <fieldset className="scheduler-border">
         <legend className="scheduler-border">Upload Addon Package:</legend>
         <div className="row">
           <div className="col-xs-12">
-            <div className="col-xs-6 col-sm-8 col-md-8 col-lg-8">
+            <div className={classesToUse}>
```
Comment (kingisaac95) [inline app/js/components/homePage/manageApps/AddAddon.jsx:25]: Good job with this 😄

**Comment by kingisaac95** [comment]
The code looks pretty good. Well done @andela-jomadoye

File: app/js/components/homePage/manageApps/AddAddon.jsx (hunk 2 above)
Comment (jomadoye) [reply app/js/components/homePage/manageApps/AddAddon.jsx:25]: Thanks for the help!

**Pull request merged**